src/invite/InviteAndSendModal.tsx

```diff
@@ -18,13 +17,14 @@ interface Props {
 export default function InviteAndSendModal({ isVisible, name, onInvite, onCancel }: Props) {
   const { t } = useTranslation()
   const rewardAmount = useSelector(inviteRewardCusdSelector)
+  const inviteRewardsEnabled = useSelector(inviteRewardsActiveSelector)
 
   const { IS_IN_EUROPE } = useCountryFeatures()
   const currency = IS_IN_EUROPE ? Currency.Euro : Currency.Dollar
 
   return (
     <Dialog
-      title={t('inviteAndSendDialog.title', { name, appName: APP_NAME })}
```

Comment:
@lisaqyli i had removed this because `appName` is global [here](https://github.com/valora-inc/wallet/blob/35147b04919306b190826235a9575a3693823316/src/i18n/index.ts#L57) :)